.azure/lint-linux.yml

```diff
@@ -21,10 +21,12 @@ jobs:
           python -m pip install --upgrade pip setuptools wheel virtualenv
           virtualenv test-job
           source test-job/bin/activate
-          pip install -U -r requirements.txt -r requirements-dev.txt -c constraints.txt
-          pip install -U -c constraints.txt -e .
-          pip install -U "qiskit-aer" -c constraints.txt
-          pip install -e .
```

Comment:
FYI we were calling `install -e .` twice here: https://github.com/Qiskit/qiskit-terra/pull/9756